### PR TITLE
Add multiclass

### PR DESCRIPTION
**Pull request by Eleven1Liu**
**Description** 
- Add `--multiclass` and update the related files
- Test on LEDGAR

|     Macro-F1     |     Micro-F1     |
|-----------------:|-----------------:|
|     80.5773      |     86.9000      |

- after PR #203

### CODE REVIEW

**Comment by JamesLYC88** [comment]
It seems that the new check does not identify the missing `multi class` option
because before adding it to tune config, it does not show an error?

**Comment by cjlin1** [comment]
Yes this is an issue. Please check what happened

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # LibMultiLabel — a Library for Multi-label Text Classification
 
-LibMultiLabel is a simple tool with the following functionalities.
+LibMultiLabel is a library for multi-label text classification and
```
Comment (cjlin1) [inline README.md:3]:
```suggestion
LibMultiLabel is a library for binary, multi-class, and multi-label classification. It has the following functionalities
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+a simple command line tool with the following functionalities
```
Comment (cjlin1) [inline README.md:4]: remove this line

File: docs/index.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 LibMultiLabel - a Library for Multi-label Text Classification
 =============================================================
 
-LibMultiLabel is a library for multi-label text classification
+LibMultiLabel is a library for binary, multi-class, multi-label text classification
```
Comment (cjlin1) [inline docs/index.rst:4]: do similar changes!

File: libmultilabel/nn/nn_utils.py [hunk 4]
```diff
@@ -67,6 +69,7 @@ def init_model(model_name,
         weight_decay (int, optional): Weight decay factor. Defaults to 0.
         metric_threshold (float, optional): Threshold to monitor for metrics. Defaults to 0.5.
         monitor_metrics (list, optional): Metrics to monitor while validating. Defaults to None.
+        multiclass (bool, optional): Enable multiclass mode. Defaults to False.
```
Comment (Eleven1Liu) [inline libmultilabel/nn/nn_utils.py:72]: Add loss function

**Review by cjlin1 — APPROVED** [review]

**Pull request merged**